README.md

```diff
@@ -1,75 +1,71 @@
 
 # Project Name
-
-> One paragraph statement about the project.
+To Do list
+> This project for manage To do list .
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+-  HTML,CSS,JavaScript
+-  Visual Studio Code: A source code editor
+-  git: A tool for managing source code
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://mustabbas.github.io/TodoList/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+**If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
 
+Download the ZIP from this location, or run the following git command to clone the files to your machine:
 
-To get a local copy up and running follow these simple example steps.
+git clone https://github.com/mustabbas/AwesomeBooks
+Once the files are on your machine, open the course folder in Visual Studio Code.
 
-### Prerequisites
+With the files open in Visual Studio Code, press the Go Live button at the bottom of the window to launch the files with Live Server.**
 
-### Setup
 
-### Install
+To get a local copy up and running follow these simple example steps.
 
-### Usage
+### Prerequisites
+Any modern web browser will do.
 
-### Run tests
+### Setup
+Follow the steps described before.
 
-### Deployment
 
+## Authors
 
+👤 **MUSTAFA ABBAS**
 
-## Authors
+- GitHub: [@githubhandle](https://github.com/mustabbas)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mustafa-abbas-7555ba10a)
 
-👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **NAKITTO CATHERINE**
 
-👤 **Author2**
+- GitHub: [@githubhandle](https://github.com/Cathella)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/nakitto-catherine2020/)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/mustabbas/AwesomeBooks/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- I would like to thank the designer who design this website [Cindy Shin](https://www.behance.net/adagio07) [design guidelines,](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 
```

Comment:
- [ ] This acknowledgment is invalid. Cindy Shin has not participated in this project in any way.